app/models/spree/user.rb

```diff
@@ -54,14 +54,14 @@ def regenerate_reset_password_token
     end
 
     def known_users
-      if admin?
-        Spree::User.where(nil)
-      else
-        Spree::User
-          .includes(:enterprises)
-          .where("enterprises.id IN (SELECT enterprise_id FROM enterprise_roles WHERE user_id = ?)",
-                 id)
-      end
+      return Spree::User.where(nil) if admin?
+
+      Spree::User
+        .joins(:enterprises)
+        .where(
+          'enterprises.id IN (SELECT enterprise_id FROM enterprise_roles WHERE user_id = ?)',
+          id
+        )
     end
```

Comment:
I was getting this error on `rails 4-1` branch. My branch start from `rails-4-1` branch
```
  1) Spree::User known_users as an enterprise user returns a list of users which manage shared enterprises
     Failure/Error: expect(u1.known_users).to include u1, u2

     ActiveRecord::StatementInvalid:
       PG::UndefinedTable: ERROR:  missing FROM-clause entry for table "enterprises"
       LINE 1: SELECT "spree_users".* FROM "spree_users"  WHERE (enterprise...
                                                                 ^
       : SELECT "spree_users".* FROM "spree_users"  WHERE (enterprises.id IN (SELECT enterprise_id FROM enterprise_roles WHERE user_id = 71))
     # ./spec/models/spree/user_spec.rb:124:in `block (4 levels) in <top (required)>'
     # ------------------
     # --- Caused by: ---
     # PG::UndefinedTable:
     #   ERROR:  missing FROM-clause entry for table "enterprises"
     #   LINE 1: SELECT "spree_users".* FROM "spree_users"  WHERE (enterprise...
     #                                                             ^
     #   ./spec/models/spree/user_spec.rb:124:in `block (4 levels) in <top (required)>'
```